src/services/doppler-legacy-client.ts

```diff
@@ -300,7 +302,32 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     });
   }
 
+  private setNoRetryAxios() {
+    axiosRetry(this.axios, { retries: 0 });
+  }
+
+  private setRetryAxiosAndLog() {
+    axiosRetry(this.axios, {
+      retries: 3,
+      shouldResetTimeout: true,
+      retryCondition: () => {
+        return true;
+      },
+      retryDelay: (retryCount: any, error: any) => {
+        addLogEntry({
+          origin: window.location.origin,
```

Comment:
We are trying to avoid using _window_ global object, maybe you could inject `window`, `location` or `navigator` using our DI.

Reply:
Injected window user DI